pypowsybl/_pypowsybl.pyi

```diff
@@ -678,6 +678,9 @@ def add_load_active_power_action(security_analysis_context: JavaHandle, action_i
 def add_load_reactive_power_action(security_analysis_context: JavaHandle, action_id: str, load_id: str, is_relative: bool, reactive_power: float) -> None: ...
 def add_generator_active_power_action(security_analysis_context: JavaHandle, action_id: str, generator_id: str, is_relative: bool, active_power: float) -> None: ...
 def add_switch_action(security_analysis_context: JavaHandle, action_id: str, switch_id: str, open: bool) -> None: ...
+def add_phase_tap_changer_position_action(security_analysis_context: JavaHandle, action_id: str, transformer_id: str, is_relative: bool, tap_position: int) -> None: ...
+def add_ratio_tap_changer_position_action(security_analysis_context: JavaHandle, action_id: str, transformer_id: str, is_relative: bool, tap_position: int) -> None: ...
```

Comment:
I think the side is missing

Reply:
maybe there is no need ?

Reply:
Side is optional for two winding transformers but needed for three winding transformers. I think it should be supported. I will add it.

Reply:
perfect thanks